src/StackExchange.Redis/PhysicalBridge.cs

```diff
@@ -852,9 +925,51 @@ internal enum BacklogStatus : byte
             SettingIdle,
             Faulted,
         }
+
         private volatile BacklogStatus _backlogStatus;
-        private async Task ProcessBacklogAsync()
+        private async Task ProcessBacklogsAsync()
+        {
```

Comment:
name nit: `ProcessBacklogAsync` ('backlog' singular)?

Reply:
On this one I want to leave it to be clear we're handling multiple backlogs, in total - e.g. when we flush cross-connection. May be semantically incorrect sometimes, but making you think is good - I'll add comments to help here.